<?xml version="1.0"?>
<launch>
    <node name="predictor_node" pkg="detector_2d" type="predict.py" output="screen">
        <rosparam command="load" file="$(find detector_2d)/config/main_cfg.yaml"/>
        <param name="image_topic" value="/camera/rgb/image_raw" />
    </node>

</launch>
